test/functional/p2p_dns_seeds.py

```diff
@@ -17,6 +17,7 @@ def set_test_params(self):
     def run_test(self):
         self.existing_outbound_connections_test()
         self.existing_block_relay_connections_test()
+        self.init_arg_tests()
```

Comment:
The test runs quite long (much waiting). I don't think it's possible to prevent this (mocking time doesn't seem possible) but running `init_arg_tests` before `existing_outbound_connections_test` would save one 11 second wait because addrman would still be empty.

Reply:
good call! changed the ordering